alfred_protocol.py

```diff
@@ -84,7 +84,8 @@ def register_skills(alfred_instance):
     # load module containing the skill
     module_name = skills_folder + '.' + filepath.strip(".py")
     skill_module = import_module(module_name)
-    
+    if 'IGNORE' in dir(skill_module):
```

Comment:
Looks like the common way to check if a function exists in a python module is to try to load it and see if you get an error. So we shouldn't need this line.